apps/src/aichat/views/modelCustomization/SetupCustomization.tsx

```diff
@@ -32,6 +33,17 @@ const SetupCustomization: React.FunctionComponent = () => {
     state => state.aichat.currentAiCustomizations
   );
 
+  /** defaults to all models if not set */
+  const availableModelIds = useAppSelector(
+    state =>
+      (state.lab.levelProperties as AichatLevelProperties | undefined)
+        ?.aichatSettings?.availableModels ?? allModels.map(model => model.id)
+  );
+  const availableModels = allModels.filter(model =>
+    availableModelIds.includes(model.id)
+  );
```

Comment:
nit: if `availableModels` is undefined, it looks like we map `allModels` to its IDs and then just map those back to `allModels`. Would it be better to let `availableModelIds` be undefined if not set on the level, and then default to `allModels` is `availableModelIds` is undefined?

Reply:
ohh, yeah. That's better.